homeworks/react-continued-3/yana_yahello/src/lecturesDetails.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import "./App.css";
+import {useState} from 'react';
+import PropTypes from 'prop-types';
+
+function LectureDetails({date, title, lecturer, link}) {
+    LectureDetails.propTypes = {
+        date: PropTypes.string,
+        title: PropTypes.string,
+        lecturer: PropTypes.string,
+        link: PropTypes.string
+    };
+
+    const [isOpened, setState] = useState(false);
+
+    return (
+        <div>
+            <h2 onClick={() => setState(!isOpened)}>Date: {date}</h2>
```

Comment:
If your next state based on current value better to use function like `(isOpened) => !isOpened`

Reply:
Sorry, I have just tried to use this function, but nothing happens (